packages/replay-next/components/Expandable.tsx

```diff
@@ -64,13 +64,13 @@ export default function Expandable({
     const newIsOpen = !isOpen;
 
     // In case this change triggers a re-render that suspends, it should be in a transition.
-    startTransition(() => {
+    // startTransition(() => {
```

Comment:
This isn't going to be a working option for us. If something suspends inside of an expandable (likely) during a non-transition update, React will unmount the entire thing and show a fallback.